### ISSUE TITLE
ES 6 Content-Type header [] is not supported

### ISSUE THREAD

**Issue by jmrichardson** [AlexIoannides/elasticsearchr#30]
With ES6, I receive the following error when sending a query with elasticsearchr:

```
"error": "Content-Type header [] is not supported",
"status": 406
```

Apparently ES6 has [Strict Content-Type Checking for Elasticsearch REST Requests](https://www.elastic.co/blog/strict-content-type-checking-for-elasticsearch-rest-requests)

**Comment by AlexIoannides** [comment]
Thanks for pointing this out - I'll take a look ASAP.

**Comment by jmrichardson** [comment]
If it saves you any time, I forked and modified the utils.R to add headers such as:

`response <- httr::POST(rescource$search_url, body = api_call_payload, httr::add_headers("Content-Type" = "application/json"))`

Here's my forked version of[ utlis.R](https://github.com/jmrichardson/elasticsearchr/blob/master/R/utils.R)

**Comment by AlexIoannides** [comment]
Thanks Jim - would you be so kind as to send a pull request with your changes?

If there's an issue please let me know and I'll do it manually.

**Comment by AlexIoannides** [comment]
This has been fixed in v0.2.1 that has been submitted to CRAN.
